apps/ewallet/lib/ewallet/web/v1/serializers/role_serializer.ex

```diff
@@ -0,0 +1,44 @@
+defmodule EWallet.Web.V1.RoleSerializer do
+  @moduledoc """
+  Serializes roles into V1 response format.
+  """
+  alias Ecto.Association.NotLoaded
+  alias EWallet.Web.{Date, Paginator}
+  alias EWallet.Web.V1.{PaginatorSerializer, UserSerializer}
+  alias EWalletDB.Role
+
+  def serialize(%Paginator{} = paginator) do
+    PaginatorSerializer.serialize(paginator, &serialize/1)
+  end
+
+  def serialize(roles) when is_list(roles) do
+    %{
+      object: "list",
+      data: Enum.map(roles, &serialize/1)
+    }
+  end
+
+  def serialize(%Role{} = role) do
+    %{
+      object: "role",
+      id: role.id,
+      name: role.name,
+      priority: role.priority,
+      display_name: role.display_name,
+      user_ids: UserSerializer.serialize(role.users, :id),
+      users: UserSerializer.serialize(role.users),
```

Comment:
Do we really want this? I don't think we should send back users here na khrap.

Reply:
Updated. Please check.